File: app-client/src/models/CompletedDraftResponse.ts
export interface DraftPick {
    draft_id: number;
    pickNumber: number;
    name: string;
    position: string;
    team: string;
}


export interface CompletedDraftResponse {
    id: number;
    created_at: string; // timestamp
    draft_date: string; // date only
    draft_time: string; // time only
    type: string;
    uuid: string;
    picks: DraftPick[];
}

File: app-server/src/main/java/devybigboard/models/CompletedDraftResponse.java
package devybigboard.models;

import java.time.Instant;
import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;

public record CompletedDraftResponse(
        long id,
        Instant createdAt,
        LocalDate draftDate,
        LocalTime draftTime,
        String type,
        String uuid,
        List<DraftPick> picks
) {}